src/GetSitesDynamicFunctionReturnTypeExtension.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * Set return type of get_sites().
+ */
+
+declare(strict_types=1);
+
+namespace SzepeViktor\PHPStan\WordPress;
+
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\FunctionReflection;
+use PHPStan\Type\Type;
+use PHPStan\Type\ArrayType;
+use PHPStan\Type\IntegerType;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\Constant\ConstantArrayType;
+use PHPStan\Type\Constant\ConstantStringType;
+
+class GetSitesDynamicFunctionReturnTypeExtension implements \PHPStan\Type\DynamicFunctionReturnTypeExtension
+{
+    public function isFunctionSupported(FunctionReflection $functionReflection): bool
+    {
+        return in_array($functionReflection->getName(), ['get_sites'], true);
```

Comment:
```suggestion
        return $functionReflection->getName() === 'get_sites';
```

Reply:
Wanna change that in `GetPostsDynamicFunctionReturnTypeExtension` as well? That's where I copied this from

Reply:
`in_array` is for possible other functions with the same type problem.
Do as you with please!